pkg/agent/plugin/workloadattestor/systemd/systemd_posix.go

```diff
@@ -67,12 +71,29 @@ func (p *Plugin) Attest(ctx context.Context, req *workloadattestorv1.AttestReque
 	}, nil
 }
 
-func getSystemdUnitInfo(ctx context.Context, pid uint) (*DBusUnitInfo, error) {
-	conn, err := dbus.SystemBus()
+func (p *Plugin) DBusConn() (*dbus.Conn, error) {
```

Comment:
This function does not really need to be exported. We may call it `getDBusConn`.